study-builder/studies/angio/study.conf

```diff
@@ -2,7 +2,15 @@
     "tenant": {
         "domain": ${auth0.domain},
         "mgmtClientId": ${auth0.mgmtClientId},
-        "mgmtSecret": ${auth0.mgmtSecret}
+        "mgmtSecret": ${auth0.mgmtSecret},
+        "passwordPolicy": {
```

Comment:
Not really too sure where to put this. I only have this in Angio, which should be enough. Should I also add to other `study.conf` files? Will that be too redundant?

Reply:
I think we want it in all CMI studies--they'll all be on the mobile app some time :)

Reply:
Okay the funny thing here is that this is a _tenant configuration_. So once we do this for Angio, all the other CMI studies will have this because they're with the same tenant. I just wasn't sure if I should put this in all the other `study.conf` files too...

Reply:
It looks like it's at the per-db-in-tenant level in auth0.  We currently only have a single database connection, but in theory we could have > 1 per tenant.  I wonder if we shouldn't copy-paste this into the schema but instead use the auth0 [management API for the connection](https://community.auth0.com/t/getting-password-rules-for-a-given-connection/15111) and hardcode a mapping of `low`, `medium`, etc. to the structured data that the app needs.

This would probably require the [password complexity route](https://pepper-dev.datadonationplatform.org/docs/api/#operation/getStudyPasswordRequirements) to include something we can map to the connection--probably the client id would suffice.  That would punt the problem of "What if we have a client that can use two or more connections?" down the road, which seems okay.  A use case for one client with multiple db connections (that is, multiple non-oauth/social auths) doesn't leap out at me.